app/vite.config.js

```diff
@@ -35,6 +35,15 @@ export default defineConfig({
 				return data === null ? {} : undefined;
 			},
 		}),
+		{
+			name: 'watch-directus-dependencies',
+			configureServer: (server) => {
+				server.watcher.options = {
+					...server.watcher.options,
```

Comment:
Do we need to inject `...server.watcher.options` for some reason?

Reply:
Yeah, I'm overwriting the ignored files, but everything else should remain the same